packages/lodex/src/aggregateQuery.js

```diff
@@ -0,0 +1,65 @@
+import RJSON from 'relaxed-json';
+import mongoDatabase from './mongoDatabase';
+
+
+const stageNameTest = /^\$/;
+/**
+ * Take `Object` containing a MongoDB aggregate query and throw the result
+ *
+ * The input object must contain a `connectionStringURI` property, containing
+ * the connection string to MongoDB.
+ *
+ * @name LodexRunQuery
```

Comment:
This is not the right name.
```suggestion
 * @name LodexAggregateQuery
```